### ISSUE TITLE
Homebrew update grant

### ISSUE THREAD

**Issue by yorikvanhavre** [FreeCAD/FPA#271]
USD 2000 to @ipatch in two installments for grant https://github.com/FreeCAD/FPA-grant-proposals/issues/11

**Comment by yorikvanhavre** [comment]
Paid today EUR 979.46 ≃ USD 1000 for installment 01 of 02 to @ipatch via BNP transfer

**Comment by ipatch** [comment]
not sure if i need to send an email in conjunction to replying to this issue. anyways i completed the transition to qt6 the other day. i have setup to separate formulas of freecad in the homebrew-freecad tap / repo. one of the formulas builds & links against python v3.12 and qt v5. while a separate formula builds & links against python v3.12 and qt v6. both of these formulas are building freecad using the v1.0.0 tarball hosted on the main freecad repo. the CI contained within the tap (homebrew-freecad) has successfully built artifacts ie. bottles for both macos and gnu+linux. this comment is my statement for confirming the work for setting up the homebrew-freecad tap to work with qt v6 as done. if there's anything else i need to add please me know. links to the above mentioned freecad formula are provided below.

https://github.com/FreeCAD/homebrew-freecad/blob/master/Formula/freecad%401.0.0_py312.rb

https://github.com/FreeCAD/homebrew-freecad/blob/master/Formula/freecad%401.0.0_py312_qt6.rb

**Comment by yorikvanhavre** [comment]
Sorry @ipatch this came right in the middle of the holidays :sweat_smile: 

I understand the work described in your proposal is now complete?

**Comment by ipatch** [comment]
> Sorry @ipatch this came right in the middle of the holidays 😅
> 
> I understand the work described in your proposal is now complete?

yes, i created a new freecad formula for the homebrew-freecad tap ie. [freecad@1.0.0_py312_qt6](https://github.com/FreeCAD/homebrew-freecad/blob/master/Formula/freecad%401.0.0_py312_qt6.rb) which i consider complete. there is an edge case that i've described in the below issue, but my investigation so far leads me to believe it's an issue with upstream homebrew-core in relation to how the qt version 6 installation is setup via homebrew.

https://github.com/FreeCAD/homebrew-freecad/issues/666

**Comment by yorikvanhavre** [comment]
Paid today EUR 999,58 ≃ USD 1000 for installment 02 of 02 to @ipatch via BNP transfer

**Comment by ipatch** [comment]
@yorikvanhavre thank you. 🙏
